docs/articles/utilities/lease.md

```diff
@@ -0,0 +1,105 @@
+---
+uid: lease
+title: Lease
+---
+# Lease-Based Access Control System
+
+## General Definition
+
+A **lease-based access control system** is a distributed coordination mechanism in which access to a resource is granted by issuing a lease, or token, that represents temporary permission to access a resource. The lease must be acquired by any process intending to interact with the resource. This mechanism often involves the following elements:
```

Comment:
It'd simplify this to say Akka.Coordination provides a generalized "distributed lock" implementation called a `Lease` that uses a unique resource identifier inside a backing store (such as Azure Blob Storage or Kubernetes Custom Resource Definitions) to only allow one current "holder" of the lease to perform an action at any given time.

Akka.NET uses leases internally inside Split Brain Resolvers (link), Cluster.Sharding, and Cluster Singletons for this purpose - and in this document you can learn how to call and create leases in your own Akka.NET applications if needed.

Reply:
Done